src/wagtail_ai/views.py

```diff
@@ -78,3 +84,41 @@ def process(request):
         return JsonResponse({"error": "An unexpected error occurred"}, status=500)
 
     return JsonResponse({"message": response})
+
+
+class PromptEditForm(forms.ModelForm):
+    """
+    Custom form for the model admin to allow users to view and edit default prompts
+    """
+
+    class Meta:
+        model = Prompt
+        fields = ["label", "description", "prompt", "method"]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if self.instance.is_default_prompt():
+            # Make the prompt field not required if it's a default prompt
+            self.fields["prompt"].required = False
+            # Populate the placeholder with the value from DEFAULT_PROMPTS
+            self.fields["prompt"].widget.attrs[
+                "placeholder"
+            ] = self.instance.get_default_prompt_value()
```

Comment:
**Question:** Why do we need this logic at all?

Reply:
_Note: the placeholder here isn't strickly necassary, I just figured it would be an intuitive way for the user to know what value is going to be provided if they leave the field blank, although now that I think about it more, it's probably not accessible and might need more context._

Normally we want to display the overriden prompt value of a default prompt if one has been saved (e.g.self.instance.prompt), otherwise we want to display the value from `DEFAULT_PROMPTS` if the prompt value is null. However in this case we want to always populate the placeholder 
to get this behaviour:

<details>
<summary>To get this placeholder when deleting a saved prompt</summary>

![2023-12-06 14 17 40](https://github.com/wagtail/wagtail-ai/assets/35934884/2cfd123e-4ea9-4aa3-986f-d7234be16d6e)

![2023-12-06 14 17 42](https://github.com/wagtail/wagtail-ai/assets/35934884/4f049be3-fb4a-4cf7-87cb-dc64d7db4e58)
</details>

<details>
<summary>Instead of this</summary>

![2023-12-06 14 18 1412](https://github.com/wagtail/wagtail-ai/assets/35934884/640d4c46-fcf9-48ec-944e-dc5e7250a281)
</details>

Reply:
I see! This is clever. I do wonder if we're abusing placeholders slightly here though - I'm not clued up on the accessibility guidelines on the use of it.

Happy to leave it as it is.

Reply:
Yeah doing a quick read, it looks like placeholders used correctly itself is controversial, I'll keep this in for now as it's something that can be reviewed Ben E at some point.